src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -457,6 +457,12 @@ static constexpr Property<Level> level{"LOG_LEVEL"};
  */
 static constexpr Property<std::string> cache_dir{"CACHE_DIR"};
 
+/**
+ * @brief The name for setting how to handle denormals.
```

Comment:
can we have extensive description as for API 1.0?

Reply:
@chenhu-wang Could you create one more PR for description?

Reply:
@ilya-lavrenov @wangleis , the extensive description is done along with Doc PR [12132](https://github.com/openvinotoolkit/openvino/pull/12132) [12127](https://github.com/openvinotoolkit/openvino/pull/12127)